<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulatorGUI</class>
 <widget class="QMainWindow" name="SimulatorGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OS Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_number_of_processes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Processes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_delay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_number_of_virtual_pages">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Virtual Pages:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_paging_delay">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Paging Delay:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_number_of_ram_pages">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of RAM Pages:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scheduler">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scheduler</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_memory_manager">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Manager</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="pagingDelay">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nRAMPages">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nVirtualPages">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nProcesses">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>0</height>
     </size>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="delay">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="scheduler">
     <item>
      <widget class="QPushButton" name="FIFO_S">
       <property name="text">
        <string>FIFO</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SJF">
       <property name="text">
        <string>SJF</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EDF">
       <property name="text">
        <string>EDF</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RoundRobin">
       <property name="text">
        <string>RoundRobin</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>171</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="memory_manager">
     <item>
      <widget class="QPushButton" name="FIFO_MM">
       <property name="text">
        <string>FIFO</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LRU">
       <property name="text">
        <string>LRU</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="quantum">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="overload">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_quantum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantum:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_overload">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Overload:</string>
    </property>
   </widget>
   <zorder>btn_next</zorder>
   <zorder>lbl_number_of_processes</zorder>
   <zorder>lbl_delay</zorder>
   <zorder>lbl_number_of_virtual_pages</zorder>
   <zorder>lbl_paging_delay</zorder>
   <zorder>lbl_number_of_ram_pages</zorder>
   <zorder>pagingDelay</zorder>
   <zorder>nRAMPages</zorder>
   <zorder>nVirtualPages</zorder>
   <zorder>nProcesses</zorder>
   <zorder>delay</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>quantum</zorder>
   <zorder>overload</zorder>
   <zorder>lbl_quantum</zorder>
   <zorder>lbl_overload</zorder>
   <zorder>lbl_scheduler</zorder>
   <zorder>lbl_memory_manager</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
